chart/values.schema.json

```diff
@@ -1864,6 +1864,14 @@
                                         "string"
                                     ],
                                     "default": 1
+                                },
+                                "minAvailable": {
+                                    "description": "Min available pods for scheduler.",
+                                    "type": [
+                                        "integer",
+                                        "string"
+                                    ],
+                                    "default": 1
```

Comment:
the variable is commented by default in the values file, why do you set default to 1 in the schema?

Reply:
Helm doesn't take the default value into account at all, but I followed the style in the file. I think it can help developers see an example of possible value.  Could you explain why we can set default values in the schema only for specified properties in the values.yaml which already have explicit values?

Reply:
Yeah I think default has no real meaning in jsonSchema, other than aid the validation in case the value is not specified and we should be fine to use it ?

Reply:
Yep, I agree with you